### PR TITLE
Baked tables use App\Model\Entity\EntityName;

### LINKED ISSUE #32: RFC Should bake add use statement to Table for its Entity?

### PR DESCRIPTION
**Pull request by sitedyno**
Would close #32 

Please let me know if I need to change/add etc.

:)

### CODE REVIEW

**Comment by jadb** [comment]
:+1:

**Comment by sitedyno** [comment]
Working on the tests.

File: src/Template/Bake/Model/table.ctp [hunk 1]
```diff
@@ -21,6 +21,7 @@ use Cake\ORM\Query;
 use Cake\ORM\RulesChecker;
 use Cake\ORM\Table;
 use Cake\Validation\Validator;
+use <%= $namespace %>\Model\Entity\<%= $entity %>;
```
Comment (lorenzo) [inline src/Template/Bake/Model/table.ctp]: Can you put this at the top of the list?

**Comment by sitedyno** [comment]
The test will fail with it at the top as well with the plugin test. Sort them in the template using php?

File: src/Template/Bake/Model/table.ctp (hunk 1 above)
Comment (sitedyno) [inline src/Template/Bake/Model/table.ctp]: Proper output for testBakeTableWithPlugin
```php
namespace ModelTest\Model\Table;       
                                       
use Cake\ORM\Query;                    
use Cake\ORM\RulesChecker;             
use Cake\ORM\Table;                    
use Cake\Validation\Validator;         
use ModelTest\Model\Entity\BakeArticle;
```

**Comment by josegonzalez** [comment]
a+ to sorting :)

**Comment by markstory** [comment]
Thanks @sitedyno

**Pull request merged**